src/Popover.js

```diff
@@ -80,18 +97,23 @@ Popover.defaultProps = {
  * @prop {string} [className]
  * @prop {string} [dataTest=dhis2-uicore-popover]
  * @prop {number} [maxWidth=360]
+ * @prop {Boolean} observePopperResize Makes the popper update position when the popper content changes size
+ * @prop {Boolean} observeReferenceResize Makes the popper update position when the reference element changes size
  * @prop {('auto'|'auto-start'|'auto-end'|'top'|'top-start'|'top-end'|'bottom'|'bottom-start'|'bottom-end'|'right'|'right-start'|'right-end'|'left'|'left-start'|'left-end')} [placement=top]
- * @prop {function} [onBackdropClick]
+ * @prop {function} [onClickOutside]
  */
 Popover.propTypes = {
     children: propTypes.node.isRequired,
     reference: elementRefPropType.isRequired,
     arrow: propTypes.bool,
     className: propTypes.string,
     dataTest: propTypes.string,
+    elevation: propTypes.string,
     maxWidth: propTypes.number,
+    observePopperResize: propTypes.bool,
+    observeReferenceResize: propTypes.bool,
```

Comment:
Should these default to `true`?

Reply:
No, I think they should default to something falsy. The app-developer can add these props if there are reasons to expect the element size will change. Also note that we do include the `eventListeners` modifier by default, which will make sure the popper updates on `window.resize` (and scroll).

Reply:
Hmmm.... this could maybe be problematic?  We use `div` scrolling rather than `window` scrolling in a few places, including the platform layout...

Reply:
> [...] updates on window.resize (and scroll)

Sorry this part of my comment is a bit ambiguous... This modifier listens to `window.resize` and _various_ scroll events, [see here](https://github.com/popperjs/popper-core/blob/master/src/modifiers/eventListeners.js#L17-L26).

But in any case, if/when we encounter problems, we can add an additional modifier to fix things.